tests/roles/backend_services/tasks/main.yaml

```diff
@@ -91,11 +95,9 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    mkdir -p tmp
-    oc kustomize base > tmp/test_deployment.yaml
-    oc apply -f tmp/test_deployment.yaml
-  args:
-    chdir: "../config"
+    oc apply -f ../config/tmp/test_deployment.yaml
+  # args:
+  #   chdir: "../config"
```

Comment:
had to stare at this for a bit. removing the kustomization tls_overrides.j2 and merging it into the openstack_control_plane. 
i just question if it is really necessary to do it in this patch where you have lots of other nfv specific/required things going on.
It is the kind of change that might benefit of its own review.

for example, does it even make sense to keep a config/base directory any more then? this file is now no longer used https://github.com/marios/data-plane-adoption/blob/main/tests/config/base/openstack_control_plane.yaml lets remove that one too (i.e. you're now always generating it from the j2)

Reply:
FWIW my instinct is the same re making this a separate PR, but i'm also thinking about timeframes and letting other patches rebase after these changes are made, so if this ends up working ok in CI or just needing minor adjustments then perhaps our best path could be to merge the "big things" fast and iterate on any leftovers / polishing?

Reply:
I agree, ideally it would be best to have this change into 2 or 3, but realistically, we should merge this when we get green runs for multinode jobs. I have a small PR that will need to be rebased on top of this to solve conflicts, I can include the cleanup of these unused files there